site/docs/components/tabs.mdx

```diff
@@ -0,0 +1,115 @@
+- Styling:
+    - Align navigation tabs and section tabs left (in left-to-right languages) and never center within a page or section.
+    - For small screens, tab items must be presented without loss of information or functionality, and without requiring scrolling in two dimensions for vertical scrolling content at a width equivalent to 320 CSS pixels and horizontal scrolling content at a height equivalent to 256 CSS pixels, as per [W3C: Understanding Success Criterion 1.4.10: Reflow](https://www.w3.org/WAI/WCAG21/Understanding/reflow.html).
+    - Limit tabs to a maximum of 8 tab items and aim for 3–6. Avoid only 2 tabs.
+    - Avoid using icons in tab items.
+    - Tab content always appears below horizontal tabs. Tab content always appears to the right of vertical tabs in left-to-right languages and to the left of vertical tabs in right-to-left languages.
+- Tab content and ordering:
+    - The first tab is selected by default. The default tab is most important use case of the page
+    - Use smart defaults for defining tabs that accelerate users to the content they care about most. For example, you might surface the most common pathway or personalized content, such as the current user's saved comments.
+    - Avoid tabs that you can "toggle off". That is, 1 tab is always selected. If you want a tab list with no selected tabs, consider instead adding a tab representing "All" the content as the first tab.
+- Navigation:
+    - A user can navigate between tabs to reveal that tab's content by clicking or tapping on the tab item. We don't support swiping between sections using the tab content area (for example, using edge swipes on mobile devices), especially when tabs have content of different length.
+- Labels:
+    - Don't truncate long tab text. Use short tab label text, and if it absolutely must be longer, consider wrapping the text instead of truncating.
+- Fixed or scrolling:
+    - Use fixed tabs of equal width for a small number of tab items (e.g. 3) that fit inside 320px containers or viewports.
+    - Use scrolling tabs of flexible width for all other tabs. That is, you can scroll them horizontally to see more tabs offscreen.
+- Overflow:
+    - Consider providing an action next to the tabs to access additional tabs that do not fit on screen. For example, you might use an adjacent menu list.
+- Empty state:
+    - If a tab contains no content or has an error, consider disabling or hiding it according to the [Interaction states guidelines for disabled elements](/guidelines/interaction-states#disabled-state).
+- Right-To-Left (RTL):
+    - Reverse tab order for right-to-left languages.
+- Hierarchy:
+    - You might use horizontal and vertical section tabs to distinguish multiple levels of hierarchy. That is, vertical tabs might navigate between larger page sections, while horizontal tabs might navigate between data in a specific section.
```

Comment:
This section needs re-thinking. The key thing we need is to establish the hierarchy in a way that designers and engineers can apply consistently that matches user mental models and expectations. Potentially, tabs is not the place to document this.